lib/traject/macros/iiif.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module Macros
+  # Macros for extracting Stanford Specific MODS values from Nokogiri documents
+  module IIIF
+    def grab_iiif_manifest(manifest)
+      resp = Faraday.get URI.parse(manifest)
+      ::JSON.parse(resp.body) if resp.status == 200
```

Comment:
I think `resp.success?` is the idiomatic version of `resp.status == 200` (and I think covers a couple other good-enough status codes?)